variables.tf

```
@@ -451,3 +451,12 @@ variable "bucket_key_enabled" {
   EOT
   nullable    = false
 }
+
+variable "expected_bucket_owner" {
+  type        = string
+  default     = ""
```

Have you tried using this locally yet? Using `null` for the default is usually important so the provider doesn't treat it as a real value. 

```suggestion
  default     = null
```